packages/provisioner/index.js

```diff
@@ -17,8 +16,12 @@ var provision = function(abstraction, options) {
     abstraction.ens = options.ens;
   }
 
+  if (abstraction.interfaceAdapter.tezos) {
+    abstraction.interfaceAdapter.setWallet(options);
```

Comment:
I don't think this call can happen here, at least not how this `provision` function is set up now (not async). Not sure where the best place for this is.

Reply:
So if we do not do this here, the contract is unable to find a wallet.
Is there a place where we can set up the wallet when contract is instantiated?